src/Query/Query.php

```diff
@@ -11,19 +11,31 @@ class Query
     /**
      * @param array<int, string|int|float|bool> $values
      */
-    public static function terms(string $field, array $values): TermsQuery
+    public static function terms(string $field, array $values, array $params = []): TermsQuery
```

Comment:
Please remove the params arguments in all methods of this class as it should only have mandatory arguments.

Reply:
I see, sorry I did not understand it in the first place, I will therefor remove them all from here